src/store/TranscriptionsStore.spec.js

```diff
@@ -276,6 +284,23 @@ describe('TranscriptionsStore', function () {
         expect(transcriptionsStore.activeTranscriptionIndex).toBe(1)
       })
 
+      it('should show when the transcription is locked', async function () {
+        await transcriptionsStore.checkIfLocked()
+        expect(toggleModalSpy).toHaveBeenCalled()
+      })
+
+      it('should unlock a transcription', async function () {
+        await transcriptionsStore.unlockTranscription()
+        expect(patchToveSpy).toHaveBeenCalledWith(
+          '/transcriptions/1/unlock',
+          {"headers": {"If-Unmodified-Since": "Mon, June 31, 2020"}}
+        )
+      })
+
+      it('should return if the transcription is lockedByCurrentUser', function () {
+        expect(transcriptionsStore.lockedByCurrentUser).toBe(false)
```

Comment:
Is there a test to check the conditions when this is true? Maybe this evaluates true in the it statement for `should show when the transcription is locked`?

Reply:
There is not. It will take a bit of setup to make this check but I don't mind adding it.